packages/flutter/test/material/menu_style_test.dart

```diff
@@ -50,7 +50,7 @@ void main() {
                   child: MenuTheme(
                     data: const MenuThemeData(
                       style: MenuStyle(
-                        fixedSize: MaterialStatePropertyAll<Size>(Size(100, 100)),
+                        fixedSize: MaterialStatePropertyAll<Size>(Size(178, 100)),
```

Comment:
Why are all these values changing for these tests? Did something in the ComboBox implementation require changes to the underlying style defaults?